libs/ledger-live-common/src/hw/installApp.ts

```diff
@@ -20,6 +25,7 @@ export default function installApp(
     firmwareKey: app.firmware_key,
     hash: app.hash,
   }).pipe(
+    retryWithDelay(APP_INSTALL_RETRY_DELAY, APP_INSTALL_RETRY_LIMIT),
```

Comment:
Maybe we should make sure QA (or you did ?) also tests the installation of an app from the manager ? I guess now it's also retrying 5 times if it fails (which is good 🔥)

Reply:
We tried, but it is really hard to properly QA the retries, we have to manually cause an error (e.g. by cutting the network from the device) and then remove the cause of the error before the retries are over. I did test it locally by increasing the interval and adding logs to the retries, it all seems to work as it should, but it's hard to test in staging. We did check that it didn't break anything on the installations though.